src/Neo/Hardfork.cs

```diff
@@ -12,6 +12,7 @@ namespace Neo
 {
     public enum Hardfork : byte
     {
-        HF_Aspidochelone
+        HF_Aspidochelone,
+        HF_StrictMode
```

Comment:
Shouldn't it be `Bsomething`?

Reply:
I'm not sure it's worth to name it, because we were planning to remove this hardfork after T5 removal. Or not?

Reply:
> Shouldn't it be `Bsomething`?

Which name do you think is more suitable. I think we need a more general name since #2810  will also apply on it.

Reply:
Like Basilisk, see https://github.com/neo-project/neo/pull/2749#discussion_r878971641